roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -10,6 +10,32 @@
     - ansible_distribution == 'CentOS'
     - ansible_distribution_major_version == '7'
 
+- name: 'Install centos-release-ansible29 on CentOS8'
+  yum:
+    name: centos-release-ansible-29
+    state: latest
+    update_cache: true
+  tags:
+    - packages
+  when:
+    - ansible_distribution == 'CentOS'
+    - ansible_distribution_major_version == '8'
+    - ansible_repository_enabled
```

Comment:
This will need to be added to defaults and the README. Per convention, we should name it `foreman_repositories_ansible_repository-enabled`. BUT, given that its a stand-alone set of repositories, I would prefer we just introduce a simple role `ansible_repositories` and handle all of this there. That greatly simplifies scoping and would be inline with things like https://github.com/theforeman/foreman-operations-collection/tree/develop/roles/puppet_repositories